terraform-modules/aws/qldb/main.tf

```diff
@@ -4,3 +4,40 @@ resource "aws_qldb_ledger" "this" {
   deletion_protection = var.deletion_protection
   tags                = var.tags
 }
+
+resource "aws_security_group" "qldb" {
+  name        = "qldb-${var.name}"
+  description = "qldb security group"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description      = "TLS from VPC"
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
+  }
+
+  egress {
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
```

Comment:
This might set off alarms in one of the scans that Bayrons has on 0.0.0.0/0.  I think it looks at egress as well.  Set it to the RFC1918 space.

Also can you narrow down the protocol a bit?  -1?  Really...all? (same as the ingress rule)

Reply:
Probably make the ingress and egress input parameters.

Reply:
Updated security group rules as suggested